integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java

```diff
@@ -159,29 +159,38 @@ public static boolean isQueueEmpty(String queueName) throws Exception {
      */
     public static boolean assertIfSystemLogContains(LogViewerClient logViewerClient, String expected)
             throws RemoteException, LogViewerLogViewerException {
-        boolean matchFounf = false;
+        boolean matchFound = false;
         long startTime = System.currentTimeMillis();
         LogEvent[] systemLogs;
-        while (!matchFounf && (System.currentTimeMillis() - startTime) < 10000) {
-            systemLogs = logViewerClient.getAllRemoteSystemLogs();
-            if (systemLogs != null) {
-                for (LogEvent logEvent : systemLogs) {
-                    if (logEvent.getMessage().contains(expected)) {
-                        matchFounf = true;
-                        break;
-                    }
+        while (!matchFound && (System.currentTimeMillis() - startTime) < 10000) {
+            matchFound = assertIfLogExists(logViewerClient, expected);
+        }
+        return matchFound;
+    }
+
+    private static boolean assertIfLogExists(LogViewerClient logViewerClient, String expected)
+            throws RemoteException, LogViewerLogViewerException {
+
+        LogEvent[] systemLogs;
+        systemLogs = logViewerClient.getAllRemoteSystemLogs();
+        boolean matchFound = false;
+        if (systemLogs != null) {
+            for (LogEvent logEvent : systemLogs) {
```

Comment:
let's do null check for logEvent as done in [1]

[1] https://github.com/wso2/product-ei/pull/1132/files#diff-a4dc8a5b139b6488742d6f7c9a1b319fR152

Reply:
Fixed.